### PR TITLE
mes-8131-bugFixes

### PR DESCRIPTION
**Pull request by RLCorp**
## Description
added conditional appearances to fail finalisation page

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
image
image

### CODE REVIEW

File: src/app/pages/non-pass-finalisation/non-pass-finalisation.page.ts [hunk 1]
```diff
@@ -370,4 +370,13 @@ export class NonPassFinalisationPage extends PracticeableBasePageComponent imple
       TestCategory.DM, TestCategory.D1M, TestCategory.DEM, TestCategory.D1EM,
     ]);
   };
+
+  DidTestComplete = (): boolean => {
```
Comment (matthew2564) [inline src/app/pages/non-pass-finalisation/non-pass-finalisation.page.ts]: Can we make the name of the function lower camel case please i.e. ```didTestComplete```

File: src/app/pages/non-pass-finalisation/non-pass-finalisation.page.ts [hunk 2]
```diff
@@ -370,4 +370,13 @@ export class NonPassFinalisationPage extends PracticeableBasePageComponent imple
...
+    if (this.activityCode) {
+      return isAnyOf(this.activityCode.activityCode, [
+        '1', '2', '4', '5',
```
Comment (matthew2564) [inline src/app/pages/non-pass-finalisation/non-pass-finalisation.page.ts]: As this is the ```non-pass-finalisation``` page then we would never have a 1 i.e. a Pass

Also, we have an enum in the chat found here ```src/app/shared/models/activity-codes.ts```, therefore this should become

```
 return isAnyOf(this.activityCode.activityCode, [
         ActivityCodes.FAIL, 
         ActivityCodes.FAIL_PUBLIC_SAFETY,
         ActivityCodes.FAIL_CANDIDATE_STOPS_TEST,
]);
```

File: src/app/pages/non-pass-finalisation/non-pass-finalisation.page.ts (hunk 1 above)
Comment (RLCorp) [reply src/app/pages/non-pass-finalisation/non-pass-finalisation.page.ts]: renamed in newest commit

File: src/app/pages/non-pass-finalisation/non-pass-finalisation.page.ts (hunk 2 above)
Comment (RLCorp) [reply src/app/pages/non-pass-finalisation/non-pass-finalisation.page.ts]: function altered in newest commit

**Pull request closed without merging**